pkg/virt-controller/services/template.go

```diff
@@ -1056,26 +1063,51 @@ func getNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualM
 	return
 }
 
-func getCniInterfaceList(vmi *v1.VirtualMachineInstance) (ifaceListString string, cniAnnotation string) {
+func getIfaceByName(vmi *v1.VirtualMachineInstance, name string) *v1.Interface {
```

Comment:
Just out of curiosity, why not define `getIfaceByName()` on `VirtualMachineInstance`? Wouldn't it be more convenient to use?

Reply:
@danielBelenky I am a vigorous OOP denier :) so I see no implicit value of tying a helper function to a type. Especially for one-off helper functions like this one.